colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb

```diff
@@ -0,0 +1,765 @@
+{
```

Comment:
<p>We have this code twice?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
we do